app/assets/stylesheets/modules/record.scss

```diff
@@ -2,13 +2,11 @@
 .record-title {
```

Comment:
I'm 👍 on this, with a lingering doubt that maybe we should call these classees `.clamp-1` and `.clamp-2` to maybe be more bootstrap-y?

Reply:
Yeah, had the same though, now that two of us have had the same thought, maybe it's a good idea.  I'll make the change.

Reply:
And that is done now.